custom/ewsghana/models.py

```diff
@@ -101,6 +101,23 @@ def domain_object(self):
         return Domain.get_by_name(self.domain)
 
 
+class EWSMigrationStats(models.Model):
+    products_count = models.IntegerField(default=0)
+    locations_count = models.IntegerField(default=0)
+    supply_points_count = models.IntegerField(default=0)
+    sms_users_count = models.IntegerField(default=0)
+    web_users_count = models.IntegerField(default=0)
+    domain = models.CharField(max_length=128, db_index=True)
+
+
+class EWSMigrationProblem(models.Model):
```

Comment:
can we also add a created date to this table that gets set once on inserting a row? otherwise it will be hard to tell once something gets fixed if it's still an issue